WebRoot/js/ajax/dwt/widgets/DwtListView.js

```diff
@@ -2827,7 +2827,8 @@ function() {
 DwtListView.prototype._sizeChildren =
 function(height) {
 	if (this.headerColCreated && this._listDiv && (height != Dwt.DEFAULT)) {
-		Dwt.setSize(this._listDiv, Dwt.DEFAULT, height - DwtListView.HEADERITEM_HEIGHT);
```

Comment:
No, it is still used at some of the places, for instance during initializations and I doubt removing it may be problematic.